server.js

```diff
@@ -521,17 +793,30 @@ function setRoles() {
     }
 
     hitler = shPlayers[hIndex];
+    if (bots.includes(hitler)){
+        hitlerBots.push(hitler);
+    }
```

Comment:
adds hitlerbot to the hitlerbots array. does nothing now, but will be helpful for future algorithm updates